src/main/resources/sessionLinksRecoveryEmailTemplate-feedbackSessionAccessLinksNone.html

```diff
@@ -0,0 +1,25 @@
+<p>Hello User,</p>
```

Comment:
Hello ${userName},

You are receiving this email because an access link recovery request for feedback sessions related to this email address ${userEmail} was made on TEAMMATES.

You do not have any feedback sessions for the past 180 days.

If a feedback session that you expect to see is missing, it may be associated with a different email address or it is closed and result not visible for the past 180 days. You can try <a href="safasdf">using a different email address</a> or contact your course instructor(s) to check whether this email address is the one used for your course enrollment on TEAMMATES. (Should we prompt them to register if they want to access more dated sessions?)